tool/caffe_model/convert.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
move it into python/ as caffe_converter.py.
users can then use it to create caffe nets and then start training by providing data.